rb/sig/lib/selenium/webdriver/common/error.rbs

```diff
@@ -10,6 +10,12 @@ module Selenium
       URLS: Hash[Symbol, String]
 
       class WebDriverError < StandardError
+        @backtrace_locations: Array[Thread::Backtrace::Location]?
+        @cause: Exception?
+
+        attr_writer backtrace_locations:  Array[Thread::Backtrace::Location]?
+        attr_writer cause: Exception?
```

Comment:
Are these not already part of StandardError?

Reply:
Oh that is a great point, thank you so much!